__tests__/actions/user.js

```diff
@@ -0,0 +1,95 @@
+import moxios from 'moxios';
```

Comment:
You are doing a lot of async calls and not waiting them to complete. You should set the second param of each `it` to a callback that will be completed after each test